### PR TITLE
feat: added location tags, SELLER collected_by, version changes to 2.0.1

### PR DESCRIPTION
**Pull request by anuj-ondc**

### CODE REVIEW

File: api/components/examples/on-demand/confirm/confirm_0.yaml [hunk 1]
```diff
@@ -38,6 +38,17 @@ value:
...
           vehicle:
             category: AUTO_RICKSHAW
+          tags:
+            - descriptor:
+                code: LOCATION
+              display: false
+              list:
+                - descriptor:
+                    code: START_AREA
+                  value: '560043'
```
Comment (92shreyansh) [inline api/components/examples/on-demand/confirm/confirm_0.yaml]: This is not pin code but a area value like Greenpark, Hauz Khas etc.

File: api/components/examples/on-demand/confirm/confirm_2.yaml [hunk 2]
```diff
@@ -38,6 +38,17 @@ value:
               type: END
```
Comment (92shreyansh) [inline api/components/examples/on-demand/confirm/confirm_2.yaml:40]: add address field in location with complete address also add it to attribute as optional

File: api/components/examples/on-demand/confirm/confirm_0.yaml (hunk 1 above)
Comment (anuj-ondc) [reply api/components/examples/on-demand/confirm/confirm_0.yaml]: resolved

File: api/components/examples/on-demand/confirm/confirm_2.yaml (hunk 2 above)
Comment (anuj-ondc) [reply api/components/examples/on-demand/confirm/confirm_2.yaml:40]: resolved.

**Pull request merged**